Manifests/ManagedPreferencesApplications/menu.nomad.login.ad.plist

```diff
@@ -317,6 +329,18 @@
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>1.4</string>
+			<key>pfm_description</key>
+			<string>If yes, migrate users</string>
+			<key>pfm_name</key>
+			<string>Migrate</string>
+			<key>pfm_title</key>
+			<string>Migrate</string>
```

Comment:
pfm_title is really only when the `pfm_name` is insufficient or not as user-friendly.  Since this is exactly the same as pfm_name you can safely remove this.